README.md

```diff
@@ -2,31 +2,379 @@
+| `System.out.println(person2.name+" is "+person2.age);` | ![person step2](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/person_step2.png)                                                                                                                                                                                                                                                                                                |
+| `person1 = person2;`                                   | ![person step3](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/person_step3.png)<br><br>The value (a memory address) stored in `person2`<br> is copied into `person1`. This results in both variables<br> referencing the same `Person` instance named "Kim".<br> The `Person` instance named "Bola" is automatically<br>deleted after it stops being referenced by a variable. |
+| `System.out.println("After reassignment:");`           | ![person step4](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/person_step4.png)                                                                                                                                                                                                                                                                                                |
+| `System.out.println(person1.name+" is "+person1.age);` | ![person step5](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/person_step5.png)                                                                                                                                                                                                                                                                                                |
+| `System.out.println(person2.name+" is "+person2.age);` | ![person step6](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/person_step6.png)                                                                                                                                                                                                                                                                                                |
+
+
+After executing the statement `person1 = person2`, both variables point at the same `Person` object,
+so the last two print statements print the same values.
+
+```text
+Bola is 26
+Kim is 41
+After reassignment: Kim is 41
+After reassignment: Kim is 41
+```
+
+While it may seem odd to have two variables pointing at the same object, this actually is a very
+common occurrence in a Java program.  For example, calling a method that take an object as a parameter
```

Comment:
Typo: "For example, calling a method that take an object as a parameter..." Should be "takes" instead of "take"

Reply:
thanks, fixed.